addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessageTest.java

```diff
@@ -0,0 +1,283 @@
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceMessage.TransceiverType._433_92MHZ_TRANSCEIVER;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.junit.Test;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+import org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceControlMessage;
+
+/**
+ * Test for RFXCom-binding
+ *
+ * @author Mike Jagdis
+ * @since 2.0.0
+ */
+public class RFXComInterfaceControlMessageTest {
+    private RFXComBridgeConfiguration emptyConfig() {
+        RFXComBridgeConfiguration configuration = new RFXComBridgeConfiguration();
+
+        configuration.enableUndecoded = false;
+        configuration.enableImagintronixOpus = false;
+        configuration.enableByronSX = false;
+        configuration.enableRSL = false;
+        configuration.enableLighting4 = false;
+        configuration.enableFineOffsetViking = false;
+        configuration.enableRubicson = false;
+        configuration.enableAEBlyss = false;
+
+        configuration.enableBlindsT1T2T3T4 = false;
+        configuration.enableBlindsT0 = false;
+        configuration.enableProGuard = false;
+//      configuration.enableFS20Packets = false;
+        configuration.enableLaCrosse = false;
+        configuration.enableHidekiUPM = false;
+        configuration.enableADLightwaveRF = false;
+        configuration.enableMertik = false;
+
+        configuration.enableVisonic = false;
+        configuration.enableATI = false;
+        configuration.enableOregonScientific = false;
+        configuration.enableMeiantech = false;
+        configuration.enableHomeEasyEU = false;
+        configuration.enableAC = false;
+        configuration.enableARC = false;
+        configuration.enableX10 = false;
+
+        return configuration;
+    }
+
+    private void testMessage(RFXComInterfaceMessage.TransceiverType transceiverType, RFXComBridgeConfiguration configuration, String data)
+            throws RFXComException {
+
+        assertArrayEquals(DatatypeConverter.parseHexBinary(data),
+            new RFXComInterfaceControlMessage(transceiverType, configuration).decodeMessage());
+    }
+
+    @Test
+    public void testUndecodedMessage() throws RFXComException {
+        RFXComBridgeConfiguration configuration = new RFXComBridgeConfiguration();
+
+        configuration.enableUndecoded = true;
+        testMessage(_433_92MHZ_TRANSCEIVER, configuration, "0D00000203530080000000000000");
+        configuration.enableUndecoded = false;
```

Comment:
If this is for resetting, you can use method which resets for example a field configuration to a fixed test value:

http://stackoverflow.com/questions/12895129/junit-before-method